lib/tasks/add_auth_token.rake

```diff
@@ -0,0 +1,13 @@
+desc 'Make a new admin'
+task add_auth_token: :environment do
+
+  User.find_each do |user|
```

Comment:
@maskedXchaotic we need to generate tokens for verified users who don't have token

Reply:
@maskedXchaotic also use tagged logging